lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -392,3 +400,55 @@ def make_paginated_api_response(results=None, count=0, num_pages=0, next_link=No
         },
         "results": results or []
     }
+
+
+class ProfileImageMixin(object):
```

Comment:
Agreed for user and storage; but its a test mixin so I would like to keep assertions there as is and rename it to `ProfileImageTestMixin`